app/views/transcribe/display_page.html.slim

```diff
@@ -45,11 +45,11 @@
 
       .toolbar_group
         -unless @preview_xml
-          =>button_tag t('.preview'), :name => 'preview', type: 'button'
+          =>button_tag t('.preview'), :name => 'preview', type: 'submit'
```

Comment:
I'm not sure that's correct. A `type=submit` button will submit the form by default, but we don't want a Preview or Autolink buttons to submit the form, probably only the Save Changes button should have the `submit` type.